Android/app/src/main/java/io/github/project_travel_mate/utilities/ContributorsFragment.java

```diff
@@ -0,0 +1,184 @@
+import static utils.Constants.USER_TOKEN;
+
+/**
+ * @author amoraitis
+ * A simple {@link Fragment} subclass.
+ * Use the {@link ContributorsFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class ContributorsFragment extends Fragment {
+
+    @BindView(R.id.android_contributors_gv)
+    ExpandableHeightGridView android_contributors_gv;
+    @BindView(R.id.server_contributors_gv)
+    ExpandableHeightGridView server_contributors_gv;
+    private String mToken;
+    private Activity mActivity;
+    private View mContributorsView;
+    private ContributorsAdapter mAndroidContributorsAdapter;
+    private ArrayList<Contributor> mAndroidContributors = new ArrayList<>();
+    private ArrayList<Contributor> mServerContributors = new ArrayList<>();
+    private ContributorsAdapter mServerContributorsAdapter;
+    private Handler mHandler;
+
+    public ContributorsFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @return A new instance of fragment ContributorsFragment.
+     */
+    public static ContributorsFragment newInstance() {
+        ContributorsFragment fragment = new ContributorsFragment();
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        mContributorsView = inflater.inflate(R.layout.fragment_contributors, container, false);
+        ButterKnife.bind(this, mContributorsView);
+        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(mActivity);
+        mToken = sharedPreferences.getString(USER_TOKEN, null);
+        mHandler = new Handler(Looper.getMainLooper());
+        mAndroidContributorsAdapter = new ContributorsAdapter(this.getContext(), mAndroidContributors);
+        android_contributors_gv.setAdapter(mAndroidContributorsAdapter);
+
+        mServerContributorsAdapter = new ContributorsAdapter(this.getContext(), mServerContributors);
+        server_contributors_gv.setAdapter(mServerContributorsAdapter);
+
+        setContributors("Travel-Mate");
+        setContributors("server");
+        android_contributors_gv.setOnItemClickListener((adapterView, view, i, l) -> startActivity(
+                new Intent(Intent.ACTION_VIEW, Uri.parse(mAndroidContributors.get(i).getUrl()))
+        ));
+        server_contributors_gv.setOnItemClickListener((adapterView, view, i, l) -> startActivity(
+                new Intent(Intent.ACTION_VIEW, Uri.parse(mServerContributors.get(i).getUrl()))
+        ));
+        return mContributorsView;
+    }
+
+    private void setContributors(String repo) {
```

Comment:
can we name it a bit more suggestive? this method is doing two things: first one to download the list and the second one to update the ui. Maybe works "getContributorsAndUpdateUI" ? Thanks

Reply:
I don't know if its necessary, because there is another method that updates the UI. Besides that, this method isnt returning anything

Reply:
MAy be just adding a comment to this function will help.